### ISSUE TITLE
You can store numbers array as pure math function:

### ISSUE THREAD

**Issue by doctor8296** [MichaelXF/js-confuser#160]
![image](https://github.com/user-attachments/assets/aff98bb2-10b7-49c3-a545-b5618765960d)

**Comment by MichaelXF** [comment]
Interesting. This is really just using the exponent as a conditional. Math formulas can be added like how [Jscrambler's does Opaque predicates](https://svebaa.github.io/personal/blog/opaque-predicates/). But still the data in the code will eventually need to get decoded so it can always be reversed engineered.

**Comment by doctor8296** [comment]
Yep, it easily reversible, but I like the idea of "pure math programming"
